### DISCUSSION TITLE
Can I run Nuclei without Internet?

### DISCUSSION THREAD

**Discussion by umar98** [projectdiscovery/nuclei#1580] [Q&A] [answered]
Hello all,

Am pretty new to Hunting. I will be on Intranet to scan some assets, is it possible using Nuclei?

**Comment by forgedhallpass** [comment, marked as answer]
Hello @umar98,

The easiest way to find out is try give it a try ;) You can do a couple of test runs, with the `-debug` flag, or using a proxy to monitor the requests and responses to understand what is happening.

The majority of the functionalities should work (except update of nuclei and the templates), yes, but for templates that require [`interactsh`](https://github.com/projectdiscovery/interactsh), you will have to set up a stand-alone server instance.
